### PR TITLE
new left and right KINOVA models

### PR DESCRIPTION
**Pull request by AurelienRenou**
added upper arm model

### CODE REVIEW

File: set_kinova_in_ulna.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+Rototrans_matrix_world_ulna = biorbd_model.globalJCS(Q, 30).to_array()
+Rototrans_matrix_world_support = biorbd_model.globalJCS(Q, 40).to_array()
+
+# We want to find the transformation matrix btw support and ulna
+# Rototrans_matrix_ulna_support = Rototrans_matrix_ulna_world * Rototrans_matrix_world_support
+
+# Rototrans_matrix_ulna_world is the inverse matrix of Rototrans_matrix_world_ulna, we use .transpose()
+Rototrans_matrix_ulna_world = biorbd_model.globalJCS(Q, 30).transpose().to_array()
+# Be aware taht Rototrans_matrix_world_ulna.transpose() would give the wrong matrix
```
Comment (Ipuch) [inline set_kinova_in_ulna.py:60]: "that"

File: set_kinova_in_ulna.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        -1.6032,
+        -0.1205,
+    ]
+)
+# Q is the vector that we get after manually placed the support into the simulation
+
+print(biorbd_model.nbQ())
+
+# In our configuration, ulna is segment 30 and support is segment 40 ( it can change if KINOVA_merge.bioMod is
+# modify) So, we want to find the rototrans ( aka homogeneous transform) of ulna and the support form local to global
```
Comment (Ipuch) [inline set_kinova_in_ulna.py:48]: "is modifed"

Comment (Ipuch) [inline set_kinova_in_ulna.py:48]: Please, replace "modify" with "modified".
And the extra space in the parenthesis "( aka homogeneous transform)"

Comment (Ipuch) [reply set_kinova_in_ulna.py:48]: "form" with "from"

File: set_kinova_in_ulna.py [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+# coordinates.
+# Position_in_world = Rototrans_matrix_world_segment * Position_in_segment
```
Comment (Ipuch) [inline set_kinova_in_ulna.py:50]: As a reminder, here is the formula to transform a point from one frame (segment) to another (world).

File: set_kinova_in_ulna.py (hunk 1 above)
Comment (Ipuch) [inline set_kinova_in_ulna.py:60]: "that"

Comment (Ipuch) [reply set_kinova_in_ulna.py:60]: .transpose() on the Rototrans object which actually inverses the rototranslation matrix.

File: set_kinova_in_ulna.py [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+# Rototrans_matrix_ulna_world is the inverse matrix of Rototrans_matrix_world_ulna, we use .transpose()
+Rototrans_matrix_ulna_world = biorbd_model.globalJCS(Q, 30).transpose().to_array()
+# Be aware taht Rototrans_matrix_world_ulna.transpose() would give the wrong matrix
+
+# Finally
+Rototrans_matrix_ulna_support = np.matmul(Rototrans_matrix_ulna_world, Rototrans_matrix_world_support)
+
+print(Rototrans_matrix_ulna_support)
+
+# we have Rototrans_matrix_ulna_support, so Position_in_ulna = Rototrans_matrix_ulna_support * Position_in_support
```
Comment (Ipuch) [inline set_kinova_in_ulna.py:67]: so, we are now able to transform the coordinates of any point in the support frame to the ulna frame with the following formula:

**Review by Ipuch — COMMENTED** [review]
Please rename the folder Geometry into "geometry".

File: models/KINOVA_arm_reverse_left.bioMod [hunk 5]
```diff
@@ -21,7 +21,7 @@ segment part7
         -0.00019222  0.00098183  -0.00089959
         0.00001186  -0.00089959  0.00240229
     com 0.02748935 -0.03834733 0.05151232
-    meshfile geom/Reverse_Piece7_left.STL
+    meshfile Geometry/Reverse_Piece7_left.STL
```
[the remark this answers is not in the document]
Comment (AurelienRenou) [reply models/KINOVA_arm_reverse_left.bioMod:24]: done

**Comment by AurelienRenou** [comment]
done

**Review by Ipuch — COMMENTED** [review]
LGTM :+1:

**Pull request merged**